charts/scalardb/templates/config-map.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: scalardb-config-file
+data:
+  # Create a database.properties file which is config file of Scalar DB server.
+  database.properties: |-
+    scalar.db.server.port={{.Values.scalardb.server.port}}
+    scalar.db.server.prometheus_http_endpoint_port={{.Values.scalardb.server.prometheus_http_endpoint_port}}
+    scalar.db.contact_points={{.Values.scalardb.contactPoints}}
+    scalar.db.contact_port={{.Values.scalardb.contactPort}}
+    scalar.db.username={{.Values.scalardb.username}}
+    scalar.db.password={{.Values.scalardb.password}}
```

Comment:
It's now written to a file through ConfigMap, but I think it should be done through Secret.

@superbrothers 
Can we do it easily without changing Scalar DB?

Reply:
I had missed here. Passing credentials through ConfigMap is a security issue. 

Can ScalarDB use environment variables to set username and password like ledger?

https://github.com/scalar-labs/helm-charts/blob/cd59731caf1f2deb4cf926fd06683b0903d9c9bb/charts/scalardl/templates/ledger/deployment.yaml#L45-L62

If it can, we can change to use a Secret object like ledger.

- https://github.com/scalar-labs/helm-charts/blob/scalardl-2.0.0/charts/scalardl/templates/ledger/secret.yaml

In addition, if it can split `database.properties` file, we can pass a username and password parts as a Secret object.

Reply:
@brfrn169 @brfrn169 Will we make Scalar DB server supporting environment variables as configurations?

Reply:
@thongdk8 This has not been addressed and the password is stored in ConfigMap. You will have to change this in another PR so that the password is stored in Secret.

cc/ @feeblefakie